README.md

```diff
@@ -49,70 +49,41 @@ messenger.on('text', ({reply, text}) => {
 messenger.start();
 ```
 
-The event name and what's in the `data` for each event handler. 
+#### Event `data`
+
+All events contain the following attributes in the `data`:
```

Comment:
I was 50/50 about doing this, but if you had the same idea, 👍 

My thought process:
* dang, it would be nice if I didn't have to repeat myself
* splitting docs makes it harder to RTFM

My conclusion was: Since this is a README: conciseness matters. So let's DRY it. Being "all in one place" is something that generated documentation can provide #51